### ISSUE TITLE
[RFC] Stash icon almost invisible on windows 10 dark mode

### ISSUE THREAD

**Issue by vallerydelexy** [stashapp/stash#3172]
*### Scope*
just the color, please...
its almost invisible

## Examples
![image](https://user-images.githubusercontent.com/17786532/203318756-c07273cf-e3c7-40e2-8f37-46b43663c01e.png)
![image](https://user-images.githubusercontent.com/17786532/203319139-5ee5bc9c-fd4f-4229-9b4a-fa4e6e88e1d3.png)

**Comment by QxxxGit** [comment]
I agree. In my system tray, it can be hard to see sometimes. I'm curious if a 1px black stroke around the icon would help.
